docker-compose.tests.yml

```diff
@@ -0,0 +1,9 @@
+services:
+  conformance_suite:
+    image: ogccite/ets-ogcapi-features10
+    depends_on: [django]
+
+  test_client:
+    build: ${PWD}/conformance-tests
```

Comment:
paths are relative in docker compose already (you can just use ./conformance-tests)